src/run_program.rs

```diff
@@ -6,10 +6,14 @@ use crate::reduction::{EvalErr, Reduction, Response};
 
 use crate::number::{ptr_from_number, Number};
 
-const QUOTE_COST: Cost = 1;
-const TRAVERSE_COST_PER_ZERO_BYTE: Cost = 1;
-const TRAVERSE_COST_PER_BIT: Cost = 1;
-const APPLY_COST: Cost = 1;
+const QUOTE_COST: Cost = 2;
+const APPLY_COST: Cost = 2;
```

Comment:
There's no way `QUOTE` takes as long as `APPLY`, is there?

Reply:
I don't have measurements for these. You're right that the apply base cost ought to be something non-trivial.

Reply:
I suppose I should try to measure the overhead of apply somehow

Reply:
Well, both quote and apply actually just update the internal stacks, but apply pushes three things and quote pushes just one (I think).

Reply:
In any case, I expect both to be asymptotically zero, so the exactly values probably don't much matter.